sovtokenfees/sovtokenfees/main.py

```diff
@@ -13,81 +26,53 @@
 from indy_common.constants import CONFIG_LEDGER_ID
 
 from plenum.common.txn_util import get_type
+from sovtokenfees.client_authnr import FeesAuthNr
+from sovtokenfees.static_fee_req_handler import StaticFeesReqHandler
+from sovtokenfees.three_phase_commit_handling import \
+    ThreePhaseCommitHandler
+from sovtoken import TOKEN_LEDGER_ID
+from sovtoken.client_authnr import TokenAuthNr
 
 
 def integrate_plugin_in_node(node):
-    from sovtokenfees.client_authnr import FeesAuthNr
-    from sovtokenfees.static_fee_req_handler import StaticFeesReqHandler
-    from sovtokenfees.three_phase_commit_handling import \
-        ThreePhaseCommitHandler
-    from sovtoken import TOKEN_LEDGER_ID
-    from sovtoken.client_authnr import TokenAuthNr
-
-    def postCatchupCompleteClb(origin_clb):
-        if origin_clb:
-            origin_clb()
-        fees_req_handler.postCatchupCompleteClbk()
-
+    token_ledger = node.db_manager.get_ledger(TOKEN_LEDGER_ID)
+    token_state = node.db_manager.get_state(TOKEN_LEDGER_ID)
     node.write_req_validator.auth_map.update(sovtokenfees_auth_map)
 
-    token_authnr = node.clientAuthNr.get_authnr_by_type(TokenAuthNr)
-    if not token_authnr:
-        raise ImportError('sovtoken plugin should be loaded, '  # noqa
-                          'authenticator not found')
-    token_req_handler = node.get_req_handler(ledger_id=TOKEN_LEDGER_ID)
-    if not token_req_handler:
-        raise ImportError('sovtoken plugin should be loaded, request '  # noqa
-                          'handler not found')
+    fees_tracker = BatchFeesTracker()
+    token_tracker = LedgerUncommittedTracker(token_state.committedHeadHash,
```

Comment:
I think, we need to move token_tracker into write_manager. It will be helpful for  postCatchupCompleteClbk too.
Please add ToDo for this.

Reply:
ok